config/locales/server.en.yml

```diff
@@ -7,6 +7,7 @@ en:
     solved_quote_length: "Number of characters to quote when displaying the solution under the first post"
     solved_topics_auto_close_hours: "Auto close topic (n) hours after the last reply once the topic has been marked as solved. Set to 0 to disable auto closing."
     show_filter_by_solved_status: "Show a dropdown to filter a topic list by solved status."
+    notify_op_on_staff_accept: "Send notification to OP when a post is marked as solution by a staff."
```

Comment:
Not to get bikesheddy here, but “OP” isn’t always clear to folks - could we say something like “the topic creator” instead?